addons/website/static/src/snippets/s_countdown/000.js

```diff
@@ -64,6 +66,25 @@ const CountdownWidget = publicWidget.Widget.extend({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * all countdowns in one line, whatever the size (S - M - L) for mobile view.
+     * adopted the parent width when the snippet(countdowns) width is too big specially
+     * when we use countdowns snippet as an inner snippet, for an example: three columns snippet
+     *
+     * @private
+     */
+    _updateCanvasSize: function () {
+        let numberOfUnits = this.display.length;
+        let parentSize = parseInt(this.$wrapper.width() / numberOfUnits);
+        this.size = parseInt(this.el.dataset.size);
+        if (!this.onlyOneUnit && (parentSize <= this.size || config.device.isMobile)) {
```

Comment:
I dont understand why you still need to know if it's a mobile here

Reply:
I also used parent width in case of circle layout with 'small' size and 'dhm' mode. that's why I used mobile here, it will render with parent width always.